### PR TITLE
Feat(content): add unfettered related hai news

### PR DESCRIPTION
**Pull request by Hurleveur**
**Content (News)**

This PR was taken from https://github.com/endless-sky/endless-sky/pull/6416 as suggested by @roadrunner56 

## Summary
Adds a few new news, one with ppl having lost it all and going to uhai space, another being more interested in whats over there (especially art-wise), and the last one about captains moving hai to unfettered space (this one happening in unfettered space).

### CODE REVIEW

File: data/hai/hai news.txt [hunk 1]
```diff
@@ -395,6 +395,73 @@ news "hai advertisement in Hai space"
 		word
 			`"`
 
+news "hai attacked by pirates"
+	location
+		government "Hai"
+	to show
+		random < 1
```
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/hai news.txt:402]: bc 2 of them say pirates attacked them but I'll make those less likely instead ur right

File: data/hai/hai news.txt [hunk 2]
```diff
@@ -395,6 +395,73 @@ news "hai advertisement in Hai space"
...
+	message
+		word
+			`"`
+		word
+			"Apparently our brothers and sisters to the north are in need of help. I may go there someday."
+			"Some very big things are going to happen in Unfettered space, and I wish to be a part of them!"
+			"I have lost everything... I knew I should not have taken that big loan. I will be heading to Unfettered space for a new life now."
+			"I come from Unfettered space. The Hai there are quite energetic and friendly once you get to know them, but I could not stand the poor living conditions."
+			"Have you heard? The art in Unfettered space is ever-changing, never dull. I wish I could go there and see it in person."
+			"Art in Unfettered space is very different from here. We could learn a lot from it."
```
Comment (EjoThims) [inline data/hai/hai news.txt:437]:
```suggestion
			"Art in Unfettered space is less stagnant than here. We could learn a lot from it."
```

Just a bit more specific and more evocative.

Alternatively could do something like "more diverse."

Or both, either in separate entires or together in one. :)

File: data/hai/hai news.txt [hunk 3]
```diff
@@ -395,6 +395,73 @@ news "hai advertisement in Hai space"
...
+			"Hai merchant"
+	to show
+		has "Unfettered Jump Drive 1: done"
+		random < 30
+	message
+		word
+			`"`
+		word
+			"So many people need relocation. I have made it my job to carry them around, asking for little if anything in return. Some are Hai going into Unfettered space because they have lost themselves."
+			"With the rumors of action soon here in the north, the demand for transportation has greatly increased. I cannot possibly fit all of them. Time to buy a new ship, I guess."
```
Comment (EjoThims) [inline data/hai/hai news.txt:456]:
```suggestion
			"With the rumors of action soon here in the north, the demand for transportation has greatly increased. I cannot possibly fit all of them. Time to buy a larger ship, I guess."
```

Again, just a bit more specific.

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/hai/hai news.txt (hunk 2 above)
Comment (Hurleveur) [reply data/hai/hai news.txt]: both
both is good ^^

**Review by OcelotWalrus — APPROVED** [review]

**Review by TheMarksman-ES — APPROVED** [review]

**Review by EjoThims — APPROVED** [review]

File: data/hai/hai news.txt [hunk 4]
```diff
@@ -395,6 +395,74 @@ news "hai advertisement in Hai space"
...
+			"Hai youth"
+	to show
+		random < 30
+	message
+		word
+			`"`
+		word
+			"Apparently our brothers and sisters to the north are in need of help. I may go there someday." 2
+			"Some very big things are going to happen in Unfettered space, and I wish to be a part of them!" 2
+			"I have lost everything... I knew I should not have taken that big loan. I will be heading to Unfettered space for a new life now." 2
```
Comment (roadrunner56) [inline data/hai/hai news.txt:434]: The last sentence of this sounds a little blunt. Perhaps:
```suggestion
			"I have lost everything... I knew I should not have taken that big loan. There is nothing left for me now except to go to Unfettered space for a new life." 2
```

**Comment by roadrunner56** [comment]
Stupid question but any reason why there's numbers at the end of some of the news phrases?

**Comment by bene-dictator** [comment]
Makes them more likely to come up. It has the same effect as duplicating the phrase that many times.

**Review by roadrunner56 — APPROVED** [review]

**Review by bene-dictator — APPROVED** [review]

**Review by W1zrad — COMMENTED** [review]
Just to confirm, have you tested and made sure `random` works as intended? I haven't seen it used very often in news, so just want to make sure.

**Comment by Hurleveur** [comment]
I just did some testing, removing all other options it shows the messages about as many times as one might expect, but I think that means it's too unlikely with the other messages, I keep cycling trough and barely see them.

I didnt take into account the fact the random number is effectively divided by the amount of available news.

**Comment by Hurleveur** [comment]
I just did some testing, removing all other options it shows the messages about as many times as one might expect, but I think that means it's too unlikely with the other messages, I keep cycling trough and barely see them.

I didnt take into account the fact the random number is effectively divided by the amount of available news.

**Comment by Hurleveur** [comment]
I just did some testing, removing all other options it shows the messages about as many times as one might expect, but I think that means it's too unlikely with the other messages, I keep cycling trough and barely see them.

I didnt take into account the fact the random number is effectively divided by the amount of available news, until now.

**Review by W1zrad — APPROVED** [review]
If you're comfortable with the frequency they're showing now with the chances re-balanced, we should be good then!

**Pull request merged**